why-promptlayer/evaluation-and-ranking.mdx

```diff
@@ -23,11 +23,11 @@ That means A/B testing is easy. Use the [Prompt Registry](/features/prompt-regis
 
 ## Scoring
 
-*Every request in PromptLayer has a “Score”. This score is an integer 0-100.*
+*Every request in PromptLayer has “Scores”. This scores are an integer 0-100.*
```

Comment:
*Every PromptLayer request can have multiple "Scores". A score is an integer from 0-100.*